src/java.base/share/classes/java/nio/channels/DatagramChannel.java

```diff
@@ -172,7 +172,7 @@ public static DatagramChannel open() throws IOException {
      * java.nio.channels.spi.SelectorProvider} object.  The channel will not be
      * connected.
      *
-     * @apiNote <a href="package-summary.html#unixdomain">Unix domain</a> sockets
+     * @apiNote {@linkplain java.net.UnixDomainSocketAddress Unix domain} sockets
```

Comment:
The parameter to this method is a ProtocolFamily and maybe this note should be say that StandardProtocolFamily#UNIX is not supported.

Reply:
Okay, reasonable suggestion. I was a bit quick integrating. I will file a new issue to update the spec as suggested.